tidb-retool-integration.md

```diff
@@ -0,0 +1,22 @@
+# TiDB and Retool Integration
```

Comment:
Each document should have a metadata section. Please add one according to the following template.
```
---
title: xx
summary: xx
---
```